course_flow/settings.py

```diff
@@ -22,7 +22,7 @@
 SECRET_KEY = "course_flow"
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = False
+DEBUG = True
```

Comment:
Make sure this is always false when you commit.

Reply:
@aahilgert maybe set-up a local_settings.py that is outside version control for these kinds of things?

Reply:
Yep, sorry, forgot to change it back on, like, all my commits. How does the local_settings.py work? Is it just effectively the same but overrides values in settings.py?

Reply:
Disregard that, googled. I'll add local settings so I don't have to toggle debug on and off